ogre/src/OgreLidarVisual.cc

```diff
@@ -149,6 +152,26 @@ void OgreLidarVisual::ClearVisualData()
 void OgreLidarVisual::SetPoints(const std::vector<double> &_points)
 {
   this->dataPtr->lidarPoints = _points;
+  this->dataPtr->pointColors.clear();
+  for (u_int64_t i = 0; i < this->dataPtr->lidarPoints.size(); i++)
+  {
+    this->dataPtr->pointColors.push_back(ignition::math::Color::White);
```

Comment:
we should keep the color consistent and set it to the same color as `Lidar/BlueRay`.